woocommerce/woocommerce-android
# Add bottomsheet to final screen

Part of #12707

### Description
This PR adds the Shipment Details BottomSheet to the purchase screen. The changes here refactor the shipment details so that the view receives a shipping rate summary and a readable state parameter. All the information needed to fill out the button sheet has been received from the previous screen, but we plan to refactor this behavior in the next milestones.

### Testing information
1. Open the main Shipping Label screen
2. Complete the Shipping Label creation form
3. Hit the Purchase button
4. Verify that the Shipping details bottom sheet is present and that the information is accurate

### The tests that have been performed

1. Checking that the UI is working as expected

### Images/gif

https://github.com/user-attachments/assets/ef2d1fc5-2a7f-485b-93b5-40750e053cb6



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:


<!-- Pull request guidelines: https://github.com/woocommerce/woocommerce-android/blob/develop/docs/pull-request-guidelines.md --
